src/mgr/MDSPerfMetricTypes.h

```diff
@@ -346,6 +363,17 @@ struct MDSPerfMetrics {
     denc(v.group_packed_performance_counters, p);
     DENC_FINISH(p);
   }
+  void dump(ceph::Formatter *f) const {
+    f->dump_stream("performance_counter_descriptors") << performance_counter_descriptors;
+    f->open_array_section("group_packed_performance_counters");
+    for (auto &i : group_packed_performance_counters) {
+      f->open_object_section("group_packed_performance_counters");
```

Comment:
why are these two sections named the same ?

Reply:
going to remove the inner sections, no need that